woocommerce/woocommerce-ios
# Widgets: Fix version override in extension settings

## Description

This PR removes Xcode warning

```
The CFBundleVersion of an app extension ('1') must match that of its containing parent app ('10.3.0.0').
```

## How

Extension was overriding `CURRENT_PROJECT_VERSION` value, but only for debug configuration. There is no need for that, so overriding settings was removed. It will inherit value from [Version.Public.xcconfig](https://github.com/woocommerce/woocommerce-ios/blob/a8860c2815e71278b247954e624db3ef73af8546/config/Version.Public.xcconfig)

## Testing instructions

1. Checkout and compile.
2. Make sure there is no above warning.

---
